virttest/utils_test/libvirt.py

```diff
@@ -855,7 +855,8 @@ def cleanup_pool(self, pool_name, pool_type, pool_target, emulated_image,
                 pool_target = os.path.join(self.tmpdir, pool_target)
                 if os.path.exists(pool_target):
                     shutil.rmtree(pool_target)
-            if pool_type == "gluster" or source_format == 'glusterfs':
+            manage_gluster = kwargs.get("remote_gluster_manage", "yes") == "yes"
+            if pool_type == "gluster" or source_format == 'glusterfs' and manage_gluster:
```

Comment:
Hello @smitterl, honestly I've never used gluster before, neither remote gluster, so doing just python review here:

On line 943 you skip the setting for `pool_type == "netfs" && source_format == 'glusterfs'` but here you skip it for `pool_type == "gluster"`, which doesn't look consistent to me especially when the `pool_type == "gluster"` setup (line 1025) doesn't contain the skip. Are you sure you wanted to include it there? Or can you please elaborate a bit on why those 2 places are treated while the other 2 places containing this function are not?

Reply:
Hi @ldoktor Thank you for reviewing this. Good point.
The intention of the ```manage_gluster``` is to skip any call to ```gluster.setup_or_cleanup_gluster```. The use case is where we can't have a local glusterfs-server (e.g. on s390x) and want to avoid putting ssh root credentials of the remote server (required by ```gluster.setup_or_cleanup_gluster```) in the test code for security reasons. (I spent some time trying to set up a user only for gluster mgmt but without success.) 
This is to make tp-libvirt/libvirt runnable on s390x, s. https://github.com/autotest/tp-libvirt/pull/2374 using a static gluster node, not managed by the test code.

Therefore, from my point of view it doesn't matter what the exact conditional is, I only want to add that switch.

Another way of saying this would be to write instead
```
if pool_type == "gluster" or source_format == 'glusterfs':
    if manage_gluster:
        gluster.setup_or_cleanup_gluster(...)
```
but I prefer to not nest the conditionals and increase complexity.
I'm not sure if this makes sense? Please, let me know.

Reply:
@ldoktor I updated the code with the proposed change.